nastaveni/nastaveni.php

```diff
@@ -191,7 +191,7 @@
 if (!defined('DEN_PRVNI_DATE')) define('DEN_PRVNI_DATE', date('Y-m-d', strtotime(PROGRAM_OD))); // první den v programu ve formátu YYYY-MM-DD
 if (!defined('DEN_PRVNI_UBYTOVANI')) define('DEN_PRVNI_UBYTOVANI', DEN_PRVNI_DATE); // datum, kterému odpovídá ubytovani_den (tabulka shop_predmety) v hodnotě 0
 if (!defined('PROGRAM_ZACATEK')) define('PROGRAM_ZACATEK', 8); // první hodina programu
-if (!defined('PROGRAM_KONEC')) define('PROGRAM_KONEC', 24); // konec programu (tuto hodinu už se nehraje)
+if (!defined('PROGRAM_KONEC')) define('PROGRAM_KONEC', 6); // konec programu (tuto hodinu už se nehraje)
```

Comment:
Přidal bych vysvětlivku ve smyslu _pokud je konec "před" začátkem, znamená to, že končí až následující den_

Reply:
@BurgrLada A ta vysvětlivka je kde?:D